test/e2e/networkpolicy_test.go

```diff
@@ -270,6 +271,165 @@ func TestDefaultDenyEgressPolicy(t *testing.T) {
 	}
 }
 
+// TestEgressToServerInCIDRBlock is a duplicate case as "should allow egress access to server in CIDR block [Feature:NetworkPolicy]",
+// while a bug exists in TOT release. This should be deleted when upstream is updated.
+// https://github.com/kubernetes/kubernetes/blob/v1.20.0-alpha.0/test/e2e/network/network_policy.go#L1365
+func TestEgressToServerInCIDRBlock(t *testing.T) {
+	data, err := setupTest(t)
```

Comment:
should we skip the test if IPv6 is not enabled in the cluster? There is no harm in running the test for IPv4-only clusters though...

Reply:
Makes sense. Let's skip for cluster without IPv6 CIDR. It will avoid duplicate tests for IPv4 cluster.